polymer_elements_config.yaml

```diff
@@ -176,9 +176,14 @@ files_to_generate:
   - google-chart/google-chart.html:
       omit_imports:
         - promise-polyfill-lite.html
-  - google-chart/charts-loader.html:
```

Comment:
Sorry, I was wrong. `chart-loader` is now `google-char-loader` and it is an element. I've restored it.